optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/drools/testgen/reproducer/TestGenCorruptedScoreReproducer.java

```diff
@@ -66,10 +66,10 @@ public boolean isReproducible(TestGenKieSessionJournal journal) {
         } catch (RuntimeException e) {
             if (e.getMessage() != null && e.getMessage().startsWith("No fact handle for ")) {
                 // this is common when removing insert of a fact that is later updated - not interesting
-                logger.debug("    Can't remove insert: {}", e.toString());
+                logger.debug("    Can't remove insert: {}", (Object) e);
```

Comment:
This is strange.  I have a small test app with which I tried out various versions of SLF4J, including this pattern:

    LOGGER.info("here is a Throwable: {}", (Object) t);

For me, using the "simple" backend, the stack trace is not printed.

I wonder if this depends on the specific backend?